cmd/root.go

```diff
@@ -110,38 +94,24 @@ func executePhases(cmd cobra.Command, progress *phases.Progress) error {
 	progress.PrimaryWorkflow = primaryWorkflow
 	progress.ProjectType = projectType
 
-	if cmd.Flags().Changed(cmdFlagKeyStack) {
-		progress.Stack = &cmdFlagStack
-	}
-	if progress.Stack == nil {
-		stack, err := phases.Stack(projectType)
-		if err != nil {
-			return err
-		}
-		progress.Stack = &stack
+	// stack
+	stack, err := phases.Stack(projectType)
+	if err != nil {
+		return err
 	}
+	progress.Stack = stack
 
-	if cmd.Flags().Changed(cmdFlagKeyAddWebhook) {
-		progress.AddWebhook = &cmdFlagAddWebhook
-	}
-	if progress.AddWebhook == nil {
-		wh, err := phases.AddWebhook()
-		if err != nil {
-			return err
-		}
-		progress.AddWebhook = &wh
+	// webhook
+	wh, err := phases.AddWebhook()
+	if err != nil {
+		return err
 	}
+	progress.AddWebhook = wh
 
-	if cmd.Flags().Changed(cmdFlagKeyAutoCodesign) {
-		progress.AutoCodesign = &cmdFlagAutoCodesign
-	}
-	if progress.AutoCodesign == nil {
-		codesign, err := phases.AutoCodesign(projectType)
-		if err != nil {
-			return err
-		}
-		_ = codesign
-		progress.AutoCodesign = nil
+	// codesign
+	_, err = phases.AutoCodesign(projectType)
```

Comment:
Is it in-progress?

Reply:
yep, most likely the ios codesigning files will not be returned, but codesigndoc will have an option to auto upload them